samples/src/main/java/com/backblaze/b2/sample/B2Sample.java

```diff
@@ -73,6 +80,31 @@ private static void mainGuts(PrintWriter writer,
             writer.println(String.format("  progress(%3.2f, %s)", percent, progress.toString()));
         };
 
+        bigHeader(writer, "create application key");
+        final String applicationKeyId;
+        {
+            final Set<B2Capability> capabilities = new HashSet<>();
+            capabilities.add(B2Capability.LIST_BUCKETS);
+            capabilities.add(B2Capability.READ_FILES);
+
+            final B2CreatedApplicationKey applicationKey =
+                    client.createKey(
+                            B2CreateKeyRequest.builder(capabilities, "testKey").build()
+                    );
+            applicationKeyId = applicationKey.getApplicationKeyId();
+            writer.println("key id: " + applicationKey.getApplicationKeyId() + "   key: " + applicationKey.getApplicationKey());
+        }
+
+        bigHeader(writer, "list application keys");
+        for (B2ApplicationKey key : client.applicationKeys()) {
+            writer.println("key id: " + key.getApplicationKeyId() + "   capabilities: " + key.getCapabilities());
+        }
+
+        bigHeader(writer, "delete application key");
+        {
+            client.deleteKey(B2DeleteKeyRequest.builder(applicationKeyId).build());
```

Comment:
you're welcome to add a convenience "deleteKey(String appKeyId)" method to the storage client interface which would create the request and call the real method in the interface.